addons/killtracker/XEH_postInit.sqf

```diff
@@ -97,7 +97,7 @@ GVAR(killCount) = 0;
         private _unitSide = [_unit] call _fnc_getSideFromConfig;
         private _killerSide = [_killer] call _fnc_getSideFromConfig;
         if ([_unitSide, _killerSide] call BIS_fnc_areFriendly) then {
-            _killInfo pushBack "<t color='#ff0000'>Friendly Fire</t>";
+            _killInfo pushBack LLSTRING(FriendlyFire);
```

Comment:
same for `<t color='#ff0000'>` and `</t>`

Reply:
```
        <Key ID="str_a3_cfgvehicles_modulefriendlyfire_f_0">
          <Original>Friendly Fire</Original>
          <Chinese>誤擊友方</Chinese>
          <French>Tir fratricide</French>
          <Spanish>Fuego aliado</Spanish>
          <Italian>Fuoco amico</Italian>
          <Polish>Ostrzał sojuszniczy</Polish>
          <Russian>Дружественный огонь</Russian>
          <German>Teambeschuss</German>
          <Czech>Střelba do vlastních!</Czech>
          <English>Friendly Fire</English>
          <Portuguese>Fogo amigo</Portuguese>
          <Korean>아군 오인사격</Korean>
          <Chinesesimp>友军误射</Chinesesimp>
          <Japanese>同士討ち</Japanese>
          <Turkish>Dost Atışı</Turkish>
        </Key>
```

Reply:
So? Should I remove tags or keep it consistent to other stringtables? Should I copy paste translations for this or use this ID?

Reply:
sorry for late reply, this is fine as people can change if the BIS wording isn't right